src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -1182,10 +1182,9 @@ INamedTypeSymbol IMethodSymbol.AssociatedAnonymousDelegate
 
         #region IMethodSymbolInternal
 
-        int IMethodSymbolInternal.CalculateLocalSyntaxOffset(int localPosition, SyntaxTree localTree)
-        {
-            return CalculateLocalSyntaxOffset(localPosition, localTree);
-        }
+        bool IMethodSymbolInternal.IsIterator => (object)IteratorElementType != null;
```

Comment:
```(object)IteratorElementType != null ``` We already have IsIterator internal property, we should delegate to that property rather than duplicating its implementation.